### ISSUE TITLE
[FEATURE] Events - add speaking/attending details to events

### ISSUE THREAD

**Issue by amandamartin-dev** [EddieHubCommunity/LinkFree#3711]
### Description

**EDIT:** This issue will only handle adding speaker details to personal profile pages.  A new issue can be opened later for the community page after #2600 is completed

Relates to #2600 

On the community events it would be nice to know which events folks are attending vs speaking so that interested folks could connect with them or discover sessions they may want to attend.  Proposal to add this as an option to events.  To keep the cards from getting too big, this could be an expandable option in the UI (open to other suggestions)

**Changes proposed:**
add "isSpeaking" or "isAttending" as a required option to events.  This will allow other folks heading to these events to connect with friends

if "isSpeaking", then a required field is "session" which would contain session details or title (there should be a limit on the length)

**Challenges:**  Due to the 1:many nature of events this may best be discussed after events are grouped in #2600 to figure out the best way to manage this in the UI on the community page.  Personal profiles will be easier in this respect.

**Comment by MitAbhay** [comment]
That's a great idea @amandamartin-dev. Can you tell me a little more about the sessions you are talking about?

**Comment by amandamartin-dev** [comment]
Sure! So, for example I just added an event for Developer Week SF which I am attending as a speaker.  So, what I would like to be able to share is perhaps the title of my talk or a very short description. Something like...

"isSpeaking": true,
"session": "Bring your own logic. Unlocking the power of SPI's"

could make it more clear perhaps by calling it sessionTitle, talkTitle, talkDetails, etc... open to ideas!

**Comment by MitAbhay** [comment]
ok, I got it!  So currently we can do this to personal profile event cards, not on the community events card, right?

**Comment by amandamartin-dev** [comment]
Yes, I feel like that would make the most sense for now.  I'll update the description/title so the focus is only on the personal evetns page for now. Good idea

**Comment by amandamartin-dev** [comment]
@MitAbhay did you want me to assign this issue to you?

**Comment by MitAbhay** [comment]
ok @amandamartin-dev, assign this to me. Actually, I was thinking about how will I implement this, and that's why I have not asked you to assign this to me.

**Comment by amandamartin-dev** [comment]
@MitAbhay no problem! If you want some help you may ask any questions here or in discord.  I will assign to you and am sure you will be successful!

**Comment by MitAbhay** [comment]
Hey @amandamartin-dev, I have tried this in this way. Is this good? 
If not suggest me some UI for the same :)
![image](https://user-images.githubusercontent.com/76659113/213929509-15818912-9002-438e-9d5f-1af7aada4f3c.png)

**Comment by amandamartin-dev** [comment]
I like it so far (but I must confess I am not a designer!). 

One thought, since for now this will only be on the individuals personal profile, is it necessary to include their name? wdyt?

**Comment by MitAbhay** [comment]
It's our choice, according to me, it's looking better with the name. Whats your thought on this?

**Comment by amandamartin-dev** [comment]
I think it's fine to leave it and if any other maintainers have thoughts to add or edits, they can comment in your PR before it is merged. Thanks so much for the details

**Comment by amandamartin-dev** [comment]
Closing this issues as it was merged
